CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 TBD
 
+## [v2.0.1](https://github.com/medibloc/panacea-java/releases/tag/v2.0.1) - 2021-08-11
+
+### Bug fixs
+- -[\#47] feat: Add getTx() and getTxs() for fetching a memo and so on
```

Comment:
Please add a `fix: Include a memo...` PR as well. Also, put this `feat: ...` message to the `Features`.

Reply:
Yes, I modified it.

Reply:
I modified it again because it was incorrect.